test/test.js

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+require('bedrock-karma');
+require('bedrock-views');
+require('./config');
```

Comment:
Is there a reason why this is happening vs using `test.config.js`?

Reply:
@mattcollier It looks like `config.js` contains the general application config and `test.config.js` contains the test specific configurations? Was that your reasoning @dlongley?

Reply:
@gannan08 OIC, it looks like one can start this with `npm start` in order to do dev work, is that right?

Reply:
@mattcollier, yes, you can start this with `npm start` to do dev work (or run it in an automated way for unit tests).

Reply:
Yup, sounds good.  The only thought I have since this is an emerging pattern...  I wonder if having `npm start` being associated with an `index.js` instead of `test.js` to more clearly dilettante the use cases.  And therefore, the `config.js` file would not be required in the `test.js` file, and therefore there's less confusion about configs.  You use this config for app/dev and that config for tests.

Reply:
thank you spell checker...  s/dilettante/delineate